asserts/snap_asserts.go

```diff
@@ -160,6 +165,11 @@ type SnapRevision struct {
 	timestamp    time.Time
 }
 
+// Series returns the series of the snap submitted to the store.
```

Comment:
I like this, but it means changing a couple of other doc comments for consistency. It turns out some still mention "snap build" anyway so I'll fix that issue at the same time.